ambari-server/src/test/python/custom_actions/TestCheckHost.py

```diff
@@ -447,3 +447,12 @@ def testUnsupportedDatabaseType(self, isfile_mock, format_mock, structured_out_m
 
     self.assertEquals(structured_out_mock.call_args[0][0], {'db_connection_check': {'message': '\'unsupported_db\' database type not supported.', 'exit_code': 1}})
 
+
+  @patch.object(OSCheck, "os_distribution", new = MagicMock(return_value = os_distro_value))
+  def testBuildUrl(self, ):
+    checkHost = CheckHost()
+
+    self.assertEquals('base_url/path', checkHost.build_url('base_url', 'path'))
+    self.assertEquals('base_url/path', checkHost.build_url('base_url/', 'path'))
+    self.assertEquals('base_url/path', checkHost.build_url('base_url/', '/path'))
+    self.assertEquals('base_url/path', checkHost.build_url('base_url/', 'path'))
```

Comment:
I guess one of the duplicate `'base_url/', 'path'` cases is intended to be `'base_url', '/path'` instead.

Reply:
@adoroszlai , nice catch. Thanks.